src/Http/Headers/src/HeaderUtilities.cs

```diff
@@ -517,6 +517,26 @@ public static string FormatNonNegativeInt64(long value)
         return ((ulong)value).ToString(NumberFormatInfo.InvariantInfo);
     }
 
+    /// <summary>
+    /// Converts the 64-bit numeric value to its equivalent string representation.
+    /// </summary>
+    /// <param name="value">
+    /// The number to convert.
+    /// </param>
+    /// <returns>
+    /// The string representation of the value of this instance, consisting of a sequence of digits ranging from 0 to 9 with no leading zeroes.
+    /// In case of negative numeric value it will have a leading minus sign.
+    /// </returns>
+    public static string FormatInt64(long value)
+    {
+        if (value == 0)
```

Comment:
`-1` is a commont too. Should it be cached in the same way as `0`?

Reply:
done